string intent_name
string utterance
IntentResponseEntity[] entities

